utils/splunk/splunk_helper.py

```diff
@@ -122,9 +122,11 @@ def _do_post(self, path, payload, request_timeout_seconds, splunk_ignore_config=
                 raise error
             self.log.warn("Received response with status {} and body {}".format(resp.status_code, resp.content))
         except Timeout as error:
-            self.log.error("Got a timeout error")
-            raise error
+            if not splunk_ignore_config:
+                self.log.error("Got a timeout error")
```

Comment:
do we want to log anything when we ignore the errors? It may be useful to have any clue in the logs

Reply:
logged a warning now.